.circleci/config.yml

```diff
@@ -86,21 +103,22 @@ workflows:
     jobs:
       - build-deploy:
            requires:
-              - test-nightly
+              - test-nightly-firefox
            filters:
              branches:
                only: master
       - build-release:
            requires:
-              - test-nightly
+              - test-nightly-firefox
            filters:
              tags:
                only: /.*/
              branches:
                ignore: /.*/
       - lint
-      - test-nightly
+      - test-nightly-firefox
       - test-release-firefox
+      - test-beta-firefox
```

Comment:
I'll give you a nickel if you can reorder these from `Nightly → Beta → Release`, or `Release → Beta → Nightly`.
I fear Nightly, Release, Beta will haunt my dreams.

Or if its this way for a reason, thats cool tool.